docs/Webpack.md

```diff
@@ -1,36 +0,0 @@
-# Webpack
-
```

Comment:
I agree that this webpack document isn't adding much value and should be removed. We should separately discuss how we will write up technical decisions (in an ADR format or otherwise) within the documentation of this application.